trino-connector/src/main/java/org/apache/gravitino/trino/connector/catalog/CatalogConnectorManager.java

```diff
@@ -60,7 +60,7 @@ public class CatalogConnectorManager {
 
   private static final int CATALOG_LOAD_FREQUENCY_SECOND = 10;
   private static final int NUMBER_EXECUTOR_THREAD = 1;
-  private static final int LOAD_METALAKE_TIMEOUT = 30;
+  private static final int LOAD_METALAKE_TIMEOUT = 60;
```

Comment:
Why do we increase this value? Is it really take so long to load metalake?

Reply:
In distribution environment.  if found create catalog it's take too long time. the function throw the timeout exception. I try to increase this value, and add some log to check the problems